src/http/response.ts

```diff
@@ -322,14 +367,19 @@ export class Response {
   public redirect(
     httpStatusCode: number,
     location: string,
-  ): { status: number; headers: Headers; body: any } {
+  ): Drash.Interfaces.ResponseOutput {
```

Comment:
Has this doc block been updated to. use. `Drash.Interfaces.ResponseOutput`?

Reply:
9d19552